examples/update_model_gauges.ipynb

```diff
@@ -194,7 +192,7 @@
    "outputs": [],
    "source": [
     "# NOTE: copy this line (without !) to your shell for more direct feedback\n",
-    "! hydromt update wflow wflow_piave_gauges -c setup_gauges --opt gauges_fn=test_stations.csv --opt basename=test-rain --opt snap_to_river=False --opt gauge_toml_header=\"['P-station']\" --opt gauge_toml_param=\"['atmosphere_water__precipitation_volume_flux']\" -v"
+    "! hydromt update wflow wflow_piave_gauges -i wflow_update_gauges_without_snapping.yml -v"
```

Comment:
Nice that you created the corresponding yaml file! Could you add a cell to print their contents though? So that the users can see what you are explaining. You can check other notebooks for examples.